app/service/auth_svc.py

```diff
@@ -77,7 +77,7 @@ def __init__(self):
         response = web.HTTPFound('/')
         if verified:
             await remember(request, response, data.get('username'))
-            return response
+            raise response
```

Comment:
what's this mean in context?

Reply:
This is to fix a deprecation warning I noticed while running tests.  Aiohttp is deprecating returning non-200 responses, and saying that they should be raised instead. REF: https://github.com/aio-libs/aiohttp/issues/2415